aries_vcx/src/common/proofs/verifier/verifier.rs

```diff
@@ -195,17 +193,17 @@ pub mod unit_tests {
                 .await
                 .unwrap();
             assert_eq!(
-                validate_indy_proof(&setup.profile, &prover_proof_json, &proof_req_json)
+                validate_indy_proof(&holder_setup.profile, &prover_proof_json, &proof_req_json)
                     .await
                     .unwrap_err()
                     .kind(),
-                AriesVcxErrorKind::VdrToolsError(405)
+                AriesVcxErrorKind::ProofRejected
```

Comment:
👍  just run into this as well while improving err handling a bit in https://github.com/hyperledger/aries-vcx/pull/721 and though you might have as well by what you mentioned on the last call - and yeah here it is :-)

Reply:
After this PR and #721, there won't be really explicit usage of `VdrToolsError` in aries-vcx or vdrtools - just will be left as variant for vdrtools errors which don't really need to be mapped to something (bunch of which should probably be removed)

Reply:
Thanks! reviewing 721 now, let's get that in first and I'll rebase and finish up this MR